index.js

```diff
@@ -58,6 +58,31 @@ function onInitHooks() {
 	)
 }
 
+function onInitCreateAdmin() {
```

Comment:
The function can be named simply `createAdmin`. Also, we've been referring to it as `superuser` all along (admins are a different thing in the context of Resin and can be confusing) and it'd be good to change the function name and messages below accordingly.

Reply:
agreed - and have switched to referencing as "superuser"